typedoc.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://typedoc.org/schema.json",
+	"categoryOrder": [
+		"Node Types",
+		"Nodes - Type Guards",
+		"Nodes - Flag Utilities",
+		"Nodes - Other Utilities",
+		"Modifier Utilities",
+		"Scope Utilities",
+		"Syntax Utilities",
+		"Compiler Options",
+		"*",
+		"Other"
+	],
+	"cleanOutputDir": true,
+	"defaultCategory": "Other",
+	"entryPointStrategy": "resolve",
```

Comment:
[Nit] This is the default, no? https://typedoc.org/guides/options/#entrypointstrategy

```suggestion
```

Same with `cleanOutputDir` I think?

Reply:
We can remove them if you like